cms/envs/test.py

```diff
@@ -129,9 +130,17 @@
     },
 }
 
-# This hack disables migrations during tests. We want to create tables directly from the models for speed.
-# See https://groups.google.com/d/msg/django-developers/PWPj3etj3-U/kCl6pMsQYYoJ.
-MIGRATION_MODULES = {app: "app.migrations_not_used_in_tests" for app in INSTALLED_APPS}
+# Earlier migrations were disabled during Python unit tests. If we enable migrations
+# these data migrations seed data that cause tests to fail. So for now they are being
+# skipped. Please do not add to this list.
+DISABLED_DATA_MIGRATIONS = [
+    'dark_lang.migrations.0002_data__enable_on_install',
+    'embargo.migrations.0002_data__add_countries',
+]
+
+os.environ['ENABLE_MIGRATIONS'] = '1'
```

Comment:
This line is just to enable migrations on Jenkins for now. We can make this change in the Jenkins config later.